.github/workflows/bench_ec2_reusable.yml

```diff
@@ -106,7 +106,9 @@ jobs:
       - uses: actions/checkout@v4
       - uses: ./.github/actions/bench
         with:
-          nix-shell: bench
+          nix-shell: ci
+          nix-cache: false
+          nix-verbose: ${{ inputs.always_terminate }}
```

Comment:
Why's that correlation between `nix-verbose` and `always_terminate`?

Reply:
There's inputs limit in GH dispatch workflow, so I can not add more inputs for bench_ec2_any now.
And I figure, if there is anything wrong, and we might set always terminate to false so that we can examinate what's wrong in ec2. 
Therefore it make sense to set nix-verbose to True to log more as well
Does that also make sense to you ?

Reply:
Doesn't the present version do the opposite?

Reply:
I'd say throw out `always_terminate` for the time being -- I've only once used it so far, and it's rare that you cannot reproduce issues by just firing up an instance and doing what was done in CI.

Reply:
oh yes, you're right, I mess up with the value..